test/core.js

```diff
@@ -1,84 +1,122 @@
 
-QUnit.module( "core" );
```

Comment:
I'm happier with this lower-space version but not sure why it happened. Did we change some config file?

Reply:
I submitted a PR that improves our ESLint setup and should resolve the issue you had here: #339. Let's get it merged first so that we don't go back & forth on code style here.

Reply:
#339 has landed so you can rebase now.